blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -575,11 +575,21 @@ def test_interactive_len(sql):
     assert len(t) == int(t.count())
 
 
-def test_sample(sql):
-    t = symbol('t', discover(sql))
-    result = compute(t.sample(n=1), sql)
+def test_sample_n(nyc):
+    t = symbol('t', discover(nyc))
+    result = compute(t.sample(n=14), nyc)
     s = odo(result, pd.DataFrame)
-    assert len(s) == 1
-    result2 = compute(t.sample(frac=0.5), sql)
+    assert len(s) == 14
```

Comment:
sorry, maybe I incorrectly assumed that we were doing a sample of the whole table which should just shuffle it. If this is a subset then I think that we should test that all the records exist in the base table as a sanity check that we are actually sampling correcectly.